py/foundation/geometry.py

```diff
@@ -237,6 +237,13 @@ def intersection(bs: Iterable['BBox']) -> Optional['BBox']:
       return None
     return BBox(ix, iy)
 
+  @staticmethod
+  def union(bs: Iterable['BBox']) -> 'BBox':
```

Comment:
Let's make sure to add a comment since union might not be clear to others regarding exactly the operation that is being done.